### PR TITLE
@felix => Add an author check around adding tests

### PR DESCRIPTION
**Pull request by orta**
Just writing this gave me some interesting ideas:

* Looking through PR comments to see if a trusted author has given a `:+1:`
* only checking for specs if the edits are within library code

### CODE REVIEW

**Comment by KrauseFx** [comment]
:white_check_mark: | No errors found
  ------------- | ------------ 



  &nbsp; | 1 Warning
  ------------- | ------------ 
    :warning: | Tests were not updated




<p align="right">
  Generated by 
    <a href="https://github.com/KrauseFx/danger">danger</a>
  on 
  <i>2016-01-07</i>
</p>

**Comment by orta** [comment]
and yeah, you need a `FelixBot` account :palm_tree: :palm_tree: :palm_tree:

**Comment by KrauseFx** [comment]
Done with https://github.com/fastlane/fastlane_core/pull/72#issuecomment-169938700 :rocket:

File: Dangerfile [hunk 1]
```diff
@@ -1,17 +1,19 @@
...
+# Verify that there have been tests?
 if files_modified.any? { |a| a.include?("spec") }
   message("Tests were updated / added")
 else
-  warn("Tests were not updated")
+  if is_trusted_author
+    # I guess we can trust us?
+    warn("Tests were not updated")
+  else
+    fail("Tests were not updated")
```
Comment (KrauseFx) [inline Dangerfile]: @orta On the other hand, when refactoring something, we probably don't want it to fail, right?

Comment (orta) [inline Dangerfile]: It's a toughie, but I think most cases you would end up touching the tests in refactoring.

Comment (orta) [inline Dangerfile]: also depends if we make it only accept merges on green

Comment (KrauseFx) [inline Dangerfile]: Agreed :+1:

**Comment by KrauseFx** [comment]
LGTM, let's resolve the merge conflict and :rocket:

**Comment by orta** [comment]
Think this fail is fixed in #27

**Comment by KrauseFx** [comment]
This is ready to merge anyway, right?

**Comment by orta** [comment]
eh, lets just leave it as is :dancers:

**Comment by orta** [comment]
0 PRs now :tada:

**Pull request closed without merging**